### ISSUE TITLE
Regression - Please include src/scss in npm package

### ISSUE THREAD

**Issue by madflow** [naver/billboard.js#760]
We just updated to 1.7.1 and broke how we extend Billboard `@import '~billboard.js/src/scss/billboard.scss';`

Please bring the `src/scss` folder back - or at least copy it to the `dist` folder.

Theming Billboard by extending the sass source is the obvious use case here.

Thanks!

**Comment by netil** [comment]
Hi @madflow, the structure of npm has been changed from the #680.
If you don't need explicitly handling from sass syntax, you can deal with `dist/billboard.css` file.

Anyway, I want hear feedback from others if the inclusion of the `src` folder is necessary or not.

Plz, show your thoughts adding `+1` reaction on this comments if you're in favor of adding `src` folder again to npm package.

**Comment by madflow** [comment]
In my opinion including the sass source at least is best practice. For example charting libraries:Chartist.js copies it to dist folder, C3 includes the src folder with scss, highcharts inludes the full source with scss. Javscript libraries like Vue or jQuery include the source. Bootstrap includes the sass source etc.

This is especially usefull with build system like Webpack and Rollup etc.

I must admit: There are not that many things to tweak in Billboard/C3 by extending the Sass source at the moment - but this is desirable in my opinion.

**Comment by netil** [comment]
@madflow, adding full `src` folder is more likely functional, rather than adding only `scss`.
(I'll be closing #762)

So, for the next release the structure of the npm package will be as:
```
/dist
/src
/types
CHANGELOG.md
CONTRIBUTING.md
LICENSE
package.json
README.md
```

**Comment by netil** [comment]
@madflow, released v1.8.0~
